### ISSUE TITLE
Initialization crash

### ISSUE THREAD

**Issue by vladimirmedyannikov** [yandexmobile/metrica-push-sdk-android#11]
Hello!
After updating yandex metrica lib to `"com.yandex.android:mobmetricalib:3.0.0"`, an error occurred during initialization `YandexMetricaPush.init(context)`.

```
java.lang.NoSuchMethodError: com.yandex.metrica.IReporter.onResumeSession
        at com.yandex.metrica.push.impl.aq.a(SourceFile:35)
        at com.yandex.metrica.push.impl.as.a(SourceFile:36)
        at com.yandex.metrica.push.impl.b.a(SourceFile:75)
        at com.yandex.metrica.push.YandexMetricaPush.init(SourceFile:139)
```

**Comment by NesterovichAlexey** [comment]
Hello! MetricaPush SDK does not support Metrica SDK 3.0.0. The next release will support.

**Comment by vladimirmedyannikov** [comment]
Ohhh...
When release new version planned?

**Comment by NesterovichAlexey** [comment]
We plan to start work on it in the near future.

**Comment by vladimirmedyannikov** [comment]
Ok, Thanks

**Comment by NesterovichAlexey** [comment]
Hi! 
AppMetrica Push SDK 1.1.0 is released, it supports AppMetrica SDK 3.0.0
